src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,104 @@
+.contact {
+  padding-top: 48px;
+
+  &__user {
+    display: flex;
+    gap: 16px;
+    flex-direction: column;
+  }
+
+  &__info {
+    box-sizing: border-box;
+
+    padding-left: 16px;
+    height: 48px;
+
+    background: #f8f8fa;
+    border: 1px solid transparent;
+    border-radius: 40px;
+    outline: none;
+
+    &::placeholder {
+      font-family: "Inter", sans-serif;
+      font-size: 14px;
+      line-height: 20px;
+    }
+
+    &:nth-child(3) {
+      padding-top: 14px;
+
+      border-radius: 16px;
+      padding-bottom: 148px;
+      overflow: hidden;
+      resize: none;
+    }
+
+    &:hover {
+      border: 1px solid #d5d7de;
+    }
+
+    &:focus {
+      border: 1px solid $color-black;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+
+    height: 56px;
+
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 27px;
+
+    border: 3px solid transparent;
+    border-radius: 40px;
+    background-color: $color-black;
+    color: white;
+
+    cursor: pointer;
+    transition: all 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties